package.json

```diff
@@ -65,7 +65,7 @@
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
     "classlist-polyfill": "^1.2.0",
-    "curl": "cujojs/curl#0.8.13",
+    "curl": "github:cujojs/curl#0.8.13",
```

Comment:
Was just reading https://docs.npmjs.com/cli/v7/configuring-npm/package-json#git-urls-as-dependencies

Am wondering if this should be `git+https` ? ie

`git+https://isaacs@github.com/npm/cli.git#0.8.13`

Reply:
I think we should just install from [npm](https://www.npmjs.com/package/curl-amd) like any other node dependency. Curiously the latest version on npm is 0.8.12 whereas 0.8.13 is available on GH releases 🤔 .